src/apps/vpn/ui/screens/settings/InformationCard.qml

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Window 2.12
+
+import Mozilla.VPN 1.0
+import compat 0.1
+import components 0.1
+
+import org.mozilla.Glean 0.30
+import telemetry 0.30
+
+Item {
```

Comment:
Are we able to remove this superfluous item and maybe just make the `Rectangle` the root?

Reply:
We'd still need a seemingly superfluous Rectangle to serve as `source` for the drop shadow [though it's unclear why just setting this to parent is unsupported](https://doc.qt.io/qt-5/qml-qtgraphicaleffects-dropshadow.html#source-prop).